workflow-templates/library-checks-and-release.yml

```diff
@@ -0,0 +1,51 @@
+name: Library checks & release
+
+on:
+  push:
+    branches:
+      - master
+      - next
+      - alpha
+      - beta
+
+env:
+  GH_TOKEN: ${{ secrets.GH_TOKEN }}
+
+jobs:
+  main:
+    name: Run checks and deploy
+    runs-on: [self-hosted, fear]
```

Comment:
i don't remember how this works exactly but, with this repo being public, couldn't anyone create an action from this and then use fear runners ? 🤔 @Typeform/frontend-architecture

Reply:
It would fail if the repo where this is used is public, I got this problem here: https://github.com/Typeform/eslint-config-typeform/pull/33

Reply:
neat sorcery!
Thanks Matt for the explanation 😄